01-Introduction and Goals.md

```diff
@@ -4,4 +4,21 @@
 
 ## Quality Goals
 
-## Stakeholders
\ No newline at end of file
+## Stakeholders & Expectations
+
+### Wikidata Product Manager
+
+- Store structured data for Wikipedia / Wikimedia projects in a central place (Initially sitelinks and infoboxes)
+- It should be possible to create lists from the stored structured data.
```

Comment:
```suggestion
- Allow adding, viewing and editing stored data via both web interface and API
- It should be possible to create lists from the stored structured data.
```

Reply:
I think the "programmatic access" is important enough to be mentioned separately and has been from the very start.

Reply:
I am not against this, however is this now not already covered by the "wikidata editor" steakeholder?

> - Ability to add new structured entity data on Wikidata via a desktop web interface.
> - Ability to edit existing structured entity data on Wikidata via a desktop web interface.
> - Ability to programmatically add new structured data to Wikidata.
> - Ability to programmatically edit existing structured data on Wikidata.

Reply:
yes